src/3p-frame.js

```diff
@@ -135,7 +135,8 @@ export function getIframe(
     // request completes.
     iframe.setAttribute('allow', 'sync-xhr \'none\';');
   }
-  if (isExperimentOn(parentWindow, 'sandbox-ads')) {
+  const excludeFromSandbox = ['facebook', 'embedly'];
+  if (isExperimentOn(parentWindow, 'sandbox-ads') && !excludeFromSandbox.includes(opt_type)) {
```

Comment:
/cc @jpettitt @cramforce tested all 3Ps and embedly and facebook were the only ones I found broken.
Will send a PR later for testing on those components ensuring we never see `sandbox` on them.